perceval/backends/core/discourse.py

```diff
@@ -267,11 +280,18 @@ class DiscourseClient(HttpClient):
 
     :param url: URL of the Discourse site
     :param api_key: Discourse API access token
-    :param archive: an archive to store/read fetched data
+    :param sleep_time: time to sleep in case
+        of connection problems
+    :param max_retries: number of max retries to a data source
+        before raising a RetryError exception
+    :param archive: collect issues already retrieved from an archive
```

Comment:
This parameter is repeated in the description below. I guess the first one is incorrect.